Adyen/Helpers/StringHelpers.swift

```diff
@@ -34,14 +34,14 @@ public extension AdyenScope where Base == String? {
 /// :nodoc:
 public extension AdyenScope where Base == String {
 
-    /// Return flag emoji if string is a country code; otherwise returns emty string.
+    /// Return flag emoji if string is a country code; otherwise returns empty string.
     var countryFlag: String? {
         guard CountryCodeValidator().isValid(base) else { return nil }
         let baseIndex = 127397
-        let usv = base.utf16
+        let unicodeScalarValue = base.utf16
```

Comment:
Was this done by the tool? Cause if so this is quite cool

Reply:
no of course not, I did it myself :-D